dotcom-rendering/src/web/lib/cardWrappers.tsx

```diff
@@ -435,7 +394,7 @@ export const Card50Media50 = ({
 			containerPalette={containerPalette}
 			headlineSize="large"
 			headlineSizeOnMobile="large"
-			imageSize="small"
+			imageSize="medium"
```

Comment:
Being more explicit about these image sizes for the top/bottom images allows us to correctly size the play icon in the card

Reply:
Until now, the image size hasn't really mattered with the image position is top/bottom